<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zapros</class>
 <widget class="QDialog" name="Zapros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Запросы</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>921</width>
     <height>43</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnBack">
      <property name="text">
       <string>Вернутся к меню админа</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Выйти из приложения</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="twOrg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>921</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCustomerInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Информация о покупателях</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalesSellers">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Продажи продавцов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSupplierFio">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск поставщиков по фамилии</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSort">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>421</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сортировка продавцов и покупателей по продаже по убыванию</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSum">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма продаж каждого товара</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMore">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоимость товаров больше введенной цены</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
